routes/search.js

```diff
@@ -3,11 +3,11 @@ const router = express.Router();
 
 const logger = require('../modules/logger');
 
-const { searchSellers } = require('../services/searchService');
+const searchController = require('../controllers/search');
```

Comment:
we can use destructuring here

Reply:
i thought since we'll be using all the controller functions of search, we better import whole module.

Reply:
ok